app/src/main/res/values/strings.xml

```diff
@@ -12,7 +12,7 @@
     <string name="label_pokemon_type">Pokemon Type</string>
     <string name="hint_pokemon_name">Enter name…</string>
     <string name="hint_pokemon_type">Enter type…</string>
-    <string name="label_pokemon_sightings">Pokemon Sighting Locations</string>
+    <string name="label_pokemon_sightings">Pokemon Sightings</string>
```

Comment:
I changed this to "Pokemon Sightings" because the 2 header texts for this page ("Saved Locations" and "Pokemon Sighting Locations") seemed too different. Now they are more similar and look better on the page. ("Saved Sightings" and "Pokemon Sightings")